src/main/scala/org/apache/spark/sql/datasketches/theta/aggregate/ThetaSketchAggUnion.scala

```diff
@@ -48,65 +54,81 @@ import org.apache.spark.sql.types.{AbstractDataType, DataType, IntegerType}
 )
 // scalastyle:on line.size.limit
 case class ThetaSketchAggUnion(
-    left: Expression,
-    right: Expression,
+    inputExpr: Expression,
+    lgKExpr: Expression,
+    seedExpr: Expression,
     mutableAggBufferOffset: Int = 0,
     inputAggBufferOffset: Int = 0)
   extends TypedImperativeAggregate[ThetaSketchWrapper]
-    with BinaryLike[Expression]
+    with TernaryLike[Expression]
```

Comment:
It didn't complain that the children weren't first, second, third?

Reply:
there is a set of overrides for this